anchor/programs/agreement/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+use anchor_lang::prelude::*;
```

Comment:
The EVM version of the agreements includes not a single agreement per contract but a list of all agreements created for a single community. The contract has a function `createAgreement` that allows any user to add another agreement to the list. 

In this different architecture proposed here, with a single agreement per account, how easy would be:
 - to list all agreements of a company
 - to list all agreements under a specific community